### PR TITLE
[DR] Fix assign policy wizard PVC selection step issue

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Fixes: https://issues.redhat.com/browse/DFBUGS-970

### CODE REVIEW

File: packages/mco/components/modals/app-manage-policies/helper/pvc-details-wizard-content.tsx [hunk 1]
```diff
@@ -73,13 +74,14 @@ const getLabelsFromSearchResult = (searchResult: SearchResult): string[] => {
 
 const getLabelsFromTags = (tags: TagsType, currIndex: number): string[] =>
   tags.reduce((acc, tag, index) => {
-    const labels: string[] = (tag?.[1] || []) as string[];
+    const labels: string[] = (tag?.[NameValueEditorPair.Value] ||
+      []) as string[];
```
Comment (SanjalKatiyar) [inline packages/mco/components/modals/app-manage-policies/helper/pvc-details-wizard-content.tsx:78]: nits: u r already doing `const labels: string[] =`
```suggestion
    const labels: string[] = tag?.[NameValueEditorPair.Value] ||
      [];
```

File: packages/mco/components/modals/app-manage-policies/helper/pvc-details-wizard-content.tsx [hunk 2]
```diff
@@ -73,13 +74,14 @@ const getLabelsFromSearchResult = (searchResult: SearchResult): string[] => {
...
     return currIndex !== index ? [...acc, ...labels] : acc;
   }, []) as string[];
 
 const getPlacementsFromTags = (tags: TagsType, currIndex: number): string[] =>
   tags.reduce((acc, tag, index) => {
-    const placement: string = tag?.[0] as string;
+    const placement: string = tag?.[NameValueEditorPair.Name] as string;
```
Comment (SanjalKatiyar) [inline packages/mco/components/modals/app-manage-policies/helper/pvc-details-wizard-content.tsx:84]: nits: already doing `const placement: string =`...
```suggestion
    const placement: string = tag?.[NameValueEditorPair.Name];
```

**Comment by alfonsomthd** [comment]
LGTM. @SanjalKatiyar to tag.

**Pull request merged**